docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -122,7 +122,16 @@ all, the full logs from Reporting will be the first place to look. In `kibana.ym
 
 [source,yaml]
 --------------------------------------------------------------------------------
-logging.root.level: all
+logging:
+  appenders:
+    console_appender:
+      type: console
+      layout:
+        type: pattern
+        highlight: true
+  root:
+    appenders: [default, console_appender]
+    level: debug
```

Comment:
nit: Is this one necessary? The only non-default setting here is `logging.root.level`, so we might not need to change it?

Reply:
I added a full config to address [Enabling debug logs in Kibana 7.17.3 not well documented](https://github.com/elastic/kibana/issues/134462) and because folks often don't follow links to other documentation.